Code_of_Conduct.md

```diff
@@ -0,0 +1,48 @@
+# Contributor Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
This one's still not rephrased. It's alright, but which parts have you rephrased, can you please tell?

Reply:
I have rephrased all the parts except this part as it contains specific words as you can see so I just changes 2/3 words in this section only